2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -6,7 +6,9 @@
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
-  // TODO complete this function
+  const date = new Date();
+  const currentTime = date.toLocaleTimeString();
+  document.body.innerHTML = `<div style = font-size:30px>${currentTime} </div>`;
 }
-
-// TODO execute `addCurrentTime` when the browser has completed loading the page
+setInterval(addCurrentTime, 1000);
```

Comment:
This could actually trigger `addCurrentTime` before `window load` event. 

One solution: put the function body of `addCurrrentTime` into an anonymous function, and put that anonymous function inside `setInterval`.

Reply:
Hi Wilgert, I modified it, but I didn't understand the difference between them.